packages/api/test/server-test.js

```diff
@@ -18,6 +18,48 @@ const assert = require('assert');
 const fetch = require('node-fetch');
 
 describe('flaky express server', () => {
+  describe('/repos', async () => {
+    it('returns a json object with the list of repositories, when you call GET on /repos', async () => {
+      const resp = await fetch('http://0.0.0.0:3000/api/repos', {
+        headers: { 'Content-Type': 'application/json' }
+      });
+      const sol = ['firstRepo', 'fourthRepo', 'secondRepo', 'thirdRepo'];
+      const respJSON = await resp.json();
+      assert.deepStrictEqual(respJSON.repoNames, sol);
+    });
+  });
+
+  describe('/githubauth', async () => {
+    it('redirects to a Github url', async () => {
+      const resp = await fetch('http://0.0.0.0:3000/githubauth', {
+        headers: { redirect: 'manual' }
+      });
+      assert.strictEqual(resp.url.includes('github.com/login'), true);
+    });
+  });
+
+  describe('/callback', async () => {
+    it('returns invalid state if state is incorrect', async () => {
+      const resp = await fetch('http://0.0.0.0:3000/api/callback?state=NONSENSE', {});
+      assert.strictEqual(resp.status, 401);
+    });
+
+    it('redirects to the flaky-dev home page when finished (regardless of how auth goes)', async () => {
+      const resp = await fetch('http://0.0.0.0:3000/api/callback?state=MUSTCHANGE', {
+        headers: { redirect: 'manual' }
+      });
+      assert.strictEqual(resp.url, 'https://flaky-dev-staging.web.app/');
+    });
+  });
+
+  describe('/sessionauthcheck', async () => {
+    // it('returns a JSON object indicating whether the user is authenticated', async () => {
```

Comment:
I would remove these tests if they're not something we want to assert against yet.